docs/01-Using-Fleet/13-Vulnerability-Processing.md

```diff
@@ -31,8 +31,9 @@ instance that does the processing.
 
 ## Setup
 
-Vulnerability checking is disabled by default. In order to enable it, you need to enable the software inventory feature 
-by setting the following environment variable:
+By default, the software inventory feature is enabled and vulnerability checking is disabled. Both must be enabled to check for vulnerabilities.
+
```

Comment:
```suggestion
You can check to see if the software inventory feature is enabled by running the `fleetctl get config` command. If `enable_software_inventory` is set to `true`, in the command's output, the software inventory feature is already enabled.
```

Can you please add the above paragraph which informs the user on how to see if software inventory is enabled.